ui/StatusQ/include/StatusQ/keychain.h

```diff
@@ -0,0 +1,56 @@
+#ifndef KEYCHAIN_H
+#define KEYCHAIN_H
+
+#include <QQuickItem>
+
+#ifdef __OBJC__
+#import <Foundation/Foundation.h>
+@class NSError;
+#import <LocalAuthentication/LocalAuthentication.h>
+@class LAContext;
+#else
+class LAContext;
+class NSError;
+#endif
+
+
+class KeychainItem : public QQuickItem {
```

Comment:
QObject is sufficient as it's not intended to be a visual element

Reply:
Please keep the file name and class name in sync according to the convention

Reply:
And it should not be, as it would most likely crash on anything outside macOS (having an empty impl)